src/triggers.js

```diff
@@ -265,7 +275,11 @@ export function getResponseObject(request, resolve, reject) {
       }
       return resolve(response);
     },
-    error: function(error) {
+    error: function (error) {
+      // ignore errors thrown in before-delete (during logout for example)
+      if (isSessionTrigger && request.triggerName === Types.beforeDelete) {
+        return resolve(undefined, error);
```

Comment:
Not sure resolve supports two arguments as it is a native promise IIRC.

Reply:
Resolve _here_ is a custom function, defined here. Its not the the promise resolve.
https://github.com/parse-community/parse-server/blob/56dc2e4a0bd230a76ff9f3a2e039b4d5a2b19d67/src/triggers.js#L560